backend/game/cards/keep_cards/energy_manipulation_cards/energy_hoarder.py

```diff
@@ -0,0 +1,17 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+from game.values import constants
+
+
+class EnergyHoarder(KeepCard):
+    def __init__(self):
+        Card.__init__(self, "Energy Hoarder", 3,
```

Comment:
You should be using super here, actually all of these cards should be calling super.